sunpy/map/tests/test_plotting.py

```diff
@@ -87,15 +88,23 @@ def test_plot_aia171_nowcsaxes(aia171_test_map):
     aia171_test_map.plot(axes=ax)
 
 
-@figure_test
+# @figure_test
 def test_rectangle_aia171(aia171_test_map):
     aia171_test_map.plot()
     bottom_left = SkyCoord(
         0 * u.arcsec, 0 * u.arcsec, frame=aia171_test_map.coordinate_frame)
+    top_right = SkyCoord(
+        100 * u.arcsec, 100 * u.arcsec, frame=aia171_test_map.coordinate_frame)
     w = 100 * u.arcsec
     h = 100 * u.arcsec
-    aia171_test_map.draw_rectangle(bottom_left, w, h)
 
+    with pytest.deprecated_call():
+        aia171_test_map.draw_rectangle(bottom_left, w, h)
+
+    assert aia171_test_map.draw_rectangle(bottom_left, w, h)[0].__str__() == aia171_test_map.draw_rectangle(bottom_left, top_right=top_right)[0].__str__()
```

Comment:
You can `str()` to do this instead. 

Anyway since, this was a figure test, it should be kept that way.

See https://docs.sunpy.org/en/latest/dev_guide/tests.html#figure-unit-tests on what figure tests are and how to write one.

You can write 2 separate tests for each cases.

Reply:
I've updated it. Thanks!

Reply:
In the link there is no mention of the hash function used to create new hashes. Could you please tell me more about it?

Reply:
Ah that is an oversight. There should be more details on how to create the hashes. 

Generally the easy way to get the hashes is to run the test and it should save it out to a file you can copy and paste from.

Reply:
This may be a noob question but how does one get the output from the test? I checked the code and it uses SHA256 and stores as a hex file. However when i used the terminal to generate the SHA256 for the image that is created by the preexisting API, I got a different hex string than the one stored in the `json` file.

Reply:
A json file is output if you run the entire figure test env from tox which will have the values.

Reply:
So I got that working and found the json file, but all the tests were failing repeatedly. Would this be because of a difference in environments?